packages/core/src/baseclient.ts

```diff
@@ -435,7 +435,8 @@ export abstract class BaseClient<O extends ClientOptions> implements Client<O> {
    */
   protected _prepareEvent(event: Event, hint: EventHint, scope?: Scope): PromiseLike<Event | null> {
     const options = this.getOptions();
-    return prepareEvent(options, event, hint, scope);
+    const integrations = Object.keys(this._integrations);
+    return prepareEvent(options, integrations, event, hint, scope);
```

Comment:
m: Hmm, could we maybe add this (optionally) add this to `EventHint`, instead of changing the API of `prepareEvent`? As that _could_ be considered public, as it is exported...

Reply:
Sounds good to me.